unitTests/ui/FacetSettingsTest.ts

```diff
@@ -254,5 +266,238 @@ export function FacetSettingsTest() {
         expect(facetSettings.close).not.toHaveBeenCalled();
       });
     });
+
+    describe('with all sections', () => {
```

Comment:
Upon making all these changes above, I noticed all tests still passed. So I added a few more tests to avoid reverting back to the same accessibility issues.